api/msam/chalicelib/connections.py

```diff
@@ -39,6 +39,25 @@ def connection_to_ddb_item(from_arn, to_arn, service, config):
     return item
 
 
+def connection_to_ddb_item(from_arn, to_arn, service, config):
+    """
+    Structure a discovered connection into a cache item.
+    """
+    arn = "{}:{}".format(from_arn, to_arn)
```

Comment:
This function does the same exact thing as before except I moved the item object generation into its own function on line 24 to avoid breaking other callers relying on this. Essentially, all it does is build up the ARN value and pass it on.
I did this because that behavior changes for certain services such as medialive inputs to channels and medialive channels to media packages.
Those services now use `connection_to_ddb_item_pl` on line 50 which is an exact replica of `connection_to_ddb_item` except it adds the pipeline number value to the ARN before passing it on.